examples/layer-browser/src/app.js

```diff
@@ -164,12 +164,17 @@ export default class App extends PureComponent {
   }
 
   _renderExampleLayer(example, settings, index) {
-    const {layer: Layer, props, getData} = example;
+    const {layer: Layer, props, getData, initialize} = example;
 
     if (getData && !props.data) {
       props.data = getData();
     }
 
+    if (initialize) {
+      initialize();
+      example.initialize = null;
```

Comment:
Changing the data structure like that is a bit hacky. Slightly better would be to have a separate variable `isInitialized` rather than nulling out functions...

Reply:
ok